nucliadb/nucliadb/ingest/tests/unit/service/test_writer.py

```diff
@@ -668,6 +668,10 @@ async def test_Index(self, writer: WriterServicer):
                 "nucliadb.ingest.service.writer.get_transaction_utility",
                 MagicMock(return_value=txn),
             ),
+            patch(
```

Comment:
Why is this needed now? Or, alternatively, why wasn't it needed before?

Reply:
This is a too-mocked-test. Before my changes, the resource was a mock and the methods to build the broker message weren't even called. Now we are using the resource to build the broker message, so there's a bunch of things that are actually accessed. As this test is almost useless and testing the ingest gRPC, I decided to mock the broker message generation